bin/cli.js

```diff
@@ -43,6 +43,7 @@ program
   .option('-e, --env-path [envFile]', 'the env file path')
   .option('--expose-metrics <bool>', 'expose /-/metrics', parseBoolean, true)
   .option('--enable-tracing <bool>', 'Enable tracing. This feature is in preview, so new releases may include breaking changes.', parseBoolean, false)
+  .option('-c, --custom-log-level <customLogLevel>', 'the name of the custom log level 35.', 'audit')
```

Comment:
being custom I'd provide no default

Reply:
I suggest to not support this configuration through the cli, but with custom options. What do you think about it? Something like https://github.com/mia-platform/lc39/blob/master/docs/service-options.md

Reply:
it makes sense; however I don't see why we should limit the feature support tbh, the development cost and maintenance are fairly low